challenge.md

```diff
@@ -6,39 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git clone 'url'
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git diff
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git checkout x
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git reset --hard commit-name
 
 5. Suppose that we wanted to look at the third-to-last commit on `master`; what command(s) would we write to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git log
```

Comment:
Not quite. `git log` will show us a history of commits, but to (non-destructively) wind back the clock and look at a previous commit, we'd want to use `git checkout`. In particular, since we want to look at the third-to-last commit on `master`, we could write `git checkout master~2` or (if we're already at the end of the master branch, just `git checkout HEAD~2`).